trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_bemt_induced_velocity.py

```diff
@@ -0,0 +1,99 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# compute_bemt_induced_velocity.py
+# 
+# Created:  Jun 2021, R. Erhard 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# package imports
+import numpy as np 
+from scipy.interpolate import interp1d
+
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift 
+def compute_bemt_induced_velocity(prop,geometry,cpts,wing_instance=None):  
+    """ This computes the velocity induced by the BEMT wake
+    on lifting surface control points
+
+    Assumptions:  
+       The wake contracts following formulation by McCormick.
+    
+    Source:   
+       Contraction factor: McCormick 1969, Aerodynamics of V/STOL Flight
+       
+    Inputs: 
+       prop        - propeller or rotor data structure             [Unitless] 
+       geometry    - SUAVE vehicle                                 [Unitless] 
+       cpts        - control points in segment                     [Unitless]
+    Properties Used:
+       N/A
+    """    
+    # extract parameters
+    VD = geometry.vortex_distribution
+    R  = prop.tip_radius
+    
+    # contraction factor by McCormick
+    if wing_instance == None:
+        print("No wing specified for wake analysis in compute_bemt_induced_velocity. Looking for main_wing.")
+        for wing in geometry.wings:
+            if isinstance(wing,"main_wing"):
```

Comment:
This syntax is going to throw an error. [The second argument needs to be the uninstantiated class type](https://docs.python.org/3/library/functions.html#isinstance).

[See this example](https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Helper_Functions/asymmetry_drag.py#L64)

Reply:
Fixed, thank you. The flag for bemt wake was off in the slipstream regression so it wasn't testing this part. I fixed that and it increased coverage back to 90.